docs/samples/python/text-analytics/example_text_analytics_entity_categories.yaml

```diff
@@ -0,0 +1,10 @@
+- name: Person
+  entity_type: NAME
+  supported_languages: [en, es, fr, de, it, zh-hans, ja, ko, pt-pt, pt-br]
+- name: Age
+  subcategory: Age
+  entity_type: AGE
+  supported_languages: [en, es, fr, de, it, zh-hans, ja, ko, pt-pt, pt-br]
+- name: InternationlBankingAccountNumber
+  entity_type: IBAN
+  supported_languages: [en, es, fr, de, it, pt-pt, pt-br]
```

Comment:
Do we want to add more / the whole list?

Reply:
Let's add at least safe harbor IMO

Reply:
What about the regional entities? only US?

Reply:
We can focus on US for now, this is the default Presidio behavior anyway.